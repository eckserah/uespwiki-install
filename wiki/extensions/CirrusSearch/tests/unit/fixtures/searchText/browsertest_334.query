vdyējūyeyafqhrqtwtfmvvbv不顾要死不活的姑娘风景如小D3：n t q h ra r n q r n q n r q r n w t n ran s g是否能Z或者 Ru 人也不发达噶分湖人奴嗯也能一年时光啊郭德纲）n蜂蜜犹如的还是创始人发布A大股东
