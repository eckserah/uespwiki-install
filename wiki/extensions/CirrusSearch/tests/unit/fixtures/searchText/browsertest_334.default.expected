{
    "description": "full_text search for 'vdy\u0113j\u016byeyafqhrqtwtfmvvbv\u4e0d\u987e\u8981\u6b7b\u4e0d\u6d3b\u7684\u59d1\u5a18\u98ce\u666f\u5982\u5c0fD3\uff1an\u2006t\u2006q\u2006h\u2006ra\u2006r\u2006n\u2006q\u2006r\u2006n\u2006q\u2006n\u2006r\u2006q\u2006r\u2006n\u2006w\u2006t\u2006n\u2006ran\u2006s\u2006g\u662f\u5426\u80fdZ\u6216\u8005 Ru \u4eba\u4e5f\u4e0d\u53d1\u8fbe\u5676\u5206\u6e56\u4eba\u5974\u55ef\u4e5f\u80fd\u4e00\u5e74\u65f6\u5149\u554a\u90ed\u5fb7\u7eb2\uff09n\u8702\u871c\u72b9\u5982\u7684\u8fd8\u662f\u521b\u59cb\u4eba\u53d1\u5e03A\u5927\u80a1\u4e1c'",
    "path": "wiki\/page\/_search",
    "params": {
        "search_type": "dfs_query_then_fetch",
        "timeout": "20s"
    },
    "query": {
        "_source": [
            "id",
            "title",
            "namespace",
            "redirect.*",
            "timestamp",
            "text_bytes"
        ],
        "fields": "text.word_count",
        "query": {
            "bool": {
                "minimum_number_should_match": 1,
                "should": [
                    {
                        "query_string": {
                            "query": "vdy\u0113j\u016byeyafqhrqtwtfmvvbv\u4e0d\u987e\u8981\u6b7b\u4e0d\u6d3b\u7684\u59d1\u5a18\u98ce\u666f\u5982\u5c0fD3\uff1an\u2006t\u2006q\u2006h\u2006ra\u2006r\u2006n\u2006q\u2006r\u2006n\u2006q\u2006n\u2006r\u2006q\u2006r\u2006n\u2006w\u2006t\u2006n\u2006ran\u2006s\u2006g\u662f\u5426\u80fdZ\u6216\u8005 Ru \u4eba\u4e5f\u4e0d\u53d1\u8fbe\u5676\u5206\u6e56\u4eba\u5974\u55ef\u4e5f\u80fd\u4e00\u5e74\u65f6\u5149\u554a\u90ed\u5fb7\u7eb2\uff09n\u8702\u871c\u72b9\u5982\u7684\u8fd8\u662f\u521b\u59cb\u4eba\u53d1\u5e03A\u5927\u80a1\u4e1c",
                            "fields": [
                                "all.plain^1",
                                "all^0.5"
                            ],
                            "auto_generate_phrase_queries": true,
                            "phrase_slop": 0,
                            "default_operator": "AND",
                            "allow_leading_wildcard": true,
                            "fuzzy_prefix_length": 2,
                            "rewrite": "top_terms_boost_1024",
                            "max_determinized_states": 500
                        }
                    },
                    {
                        "multi_match": {
                            "fields": [
                                "all_near_match^2"
                            ],
                            "query": "vdy\u0113j\u016byeyafqhrqtwtfmvvbv\u4e0d\u987e\u8981\u6b7b\u4e0d\u6d3b\u7684\u59d1\u5a18\u98ce\u666f\u5982\u5c0fD3\uff1an\u2006t\u2006q\u2006h\u2006ra\u2006r\u2006n\u2006q\u2006r\u2006n\u2006q\u2006n\u2006r\u2006q\u2006r\u2006n\u2006w\u2006t\u2006n\u2006ran\u2006s\u2006g\u662f\u5426\u80fdZ\u6216\u8005 Ru \u4eba\u4e5f\u4e0d\u53d1\u8fbe\u5676\u5206\u6e56\u4eba\u5974\u55ef\u4e5f\u80fd\u4e00\u5e74\u65f6\u5149\u554a\u90ed\u5fb7\u7eb2\uff09n\u8702\u871c\u72b9\u5982\u7684\u8fd8\u662f\u521b\u59cb\u4eba\u53d1\u5e03A\u5927\u80a1\u4e1c"
                        }
                    }
                ],
                "filter": [
                    {
                        "terms": {
                            "namespace": [
                                0,
                                1,
                                2,
                                3
                            ]
                        }
                    }
                ]
            }
        },
        "highlight": {
            "pre_tags": [
                "<span class=\"searchmatch\">"
            ],
            "post_tags": [
                "<\/span>"
            ],
            "fields": {
                "title": {
                    "type": "experimental",
                    "fragmenter": "none",
                    "number_of_fragments": 1,
                    "matched_fields": [
                        "title",
                        "title.plain"
                    ]
                },
                "redirect.title": {
                    "type": "experimental",
                    "fragmenter": "none",
                    "order": "score",
                    "number_of_fragments": 1,
                    "options": {
                        "skip_if_last_matched": true
                    },
                    "matched_fields": [
                        "redirect.title",
                        "redirect.title.plain"
                    ]
                },
                "category": {
                    "type": "experimental",
                    "fragmenter": "none",
                    "order": "score",
                    "number_of_fragments": 1,
                    "options": {
                        "skip_if_last_matched": true
                    },
                    "matched_fields": [
                        "category",
                        "category.plain"
                    ]
                },
                "heading": {
                    "type": "experimental",
                    "fragmenter": "none",
                    "order": "score",
                    "number_of_fragments": 1,
                    "options": {
                        "skip_if_last_matched": true
                    },
                    "matched_fields": [
                        "heading",
                        "heading.plain"
                    ]
                },
                "text": {
                    "type": "experimental",
                    "number_of_fragments": 1,
                    "fragmenter": "scan",
                    "fragment_size": 150,
                    "options": {
                        "top_scoring": true,
                        "boost_before": {
                            "20": 2,
                            "50": 1.8,
                            "200": 1.5,
                            "1000": 1.2
                        },
                        "max_fragments_scored": 5000
                    },
                    "no_match_size": 150,
                    "matched_fields": [
                        "text",
                        "text.plain"
                    ]
                },
                "auxiliary_text": {
                    "type": "experimental",
                    "number_of_fragments": 1,
                    "fragmenter": "scan",
                    "fragment_size": 150,
                    "options": {
                        "top_scoring": true,
                        "boost_before": {
                            "20": 2,
                            "50": 1.8,
                            "200": 1.5,
                            "1000": 1.2
                        },
                        "max_fragments_scored": 5000,
                        "skip_if_last_matched": true
                    },
                    "matched_fields": [
                        "auxiliary_text",
                        "auxiliary_text.plain"
                    ]
                }
            },
            "highlight_query": {
                "query_string": {
                    "query": "vdy\u0113j\u016byeyafqhrqtwtfmvvbv\u4e0d\u987e\u8981\u6b7b\u4e0d\u6d3b\u7684\u59d1\u5a18\u98ce\u666f\u5982\u5c0fD3\uff1an\u2006t\u2006q\u2006h\u2006ra\u2006r\u2006n\u2006q\u2006r\u2006n\u2006q\u2006n\u2006r\u2006q\u2006r\u2006n\u2006w\u2006t\u2006n\u2006ran\u2006s\u2006g\u662f\u5426\u80fdZ\u6216\u8005 Ru \u4eba\u4e5f\u4e0d\u53d1\u8fbe\u5676\u5206\u6e56\u4eba\u5974\u55ef\u4e5f\u80fd\u4e00\u5e74\u65f6\u5149\u554a\u90ed\u5fb7\u7eb2\uff09n\u8702\u871c\u72b9\u5982\u7684\u8fd8\u662f\u521b\u59cb\u4eba\u53d1\u5e03A\u5927\u80a1\u4e1c",
                    "fields": [
                        "title.plain^20",
                        "redirect.title.plain^15",
                        "category.plain^8",
                        "heading.plain^5",
                        "opening_text.plain^3",
                        "text.plain^1",
                        "auxiliary_text.plain^0.5",
                        "title^10",
                        "redirect.title^7.5",
                        "category^4",
                        "heading^2.5",
                        "opening_text^1.5",
                        "text^0.5",
                        "auxiliary_text^0.25"
                    ],
                    "auto_generate_phrase_queries": true,
                    "phrase_slop": 1,
                    "default_operator": "AND",
                    "allow_leading_wildcard": true,
                    "fuzzy_prefix_length": 2,
                    "rewrite": "top_terms_boost_1024",
                    "max_determinized_states": 500
                }
            }
        },
        "suggest": {
            "text": "vdy\u0113j\u016byeyafqhrqtwtfmvvbv\u4e0d\u987e\u8981\u6b7b\u4e0d\u6d3b\u7684\u59d1\u5a18\u98ce\u666f\u5982\u5c0fD3\uff1an\u2006t\u2006q\u2006h\u2006ra\u2006r\u2006n\u2006q\u2006r\u2006n\u2006q\u2006n\u2006r\u2006q\u2006r\u2006n\u2006w\u2006t\u2006n\u2006ran\u2006s\u2006g\u662f\u5426\u80fdZ\u6216\u8005 Ru \u4eba\u4e5f\u4e0d\u53d1\u8fbe\u5676\u5206\u6e56\u4eba\u5974\u55ef\u4e5f\u80fd\u4e00\u5e74\u65f6\u5149\u554a\u90ed\u5fb7\u7eb2\uff09n\u8702\u871c\u72b9\u5982\u7684\u8fd8\u662f\u521b\u59cb\u4eba\u53d1\u5e03A\u5927\u80a1\u4e1c",
            "suggest": {
                "phrase": {
                    "field": "suggest",
                    "size": 1,
                    "max_errors": 2,
                    "confidence": 2,
                    "real_word_error_likelihood": 0.95,
                    "direct_generator": [
                        {
                            "field": "suggest",
                            "suggest_mode": "always",
                            "max_term_freq": 0.5,
                            "min_doc_freq": 0,
                            "prefix_length": 2
                        }
                    ],
                    "highlight": {
                        "pre_tag": "<em>",
                        "post_tag": "<\/em>"
                    },
                    "smoothing": {
                        "stupid_backoff": {
                            "discount": 0.4
                        }
                    }
                }
            }
        },
        "stats": [
            "suggest",
            "full_text"
        ],
        "size": 20,
        "rescore": [
            {
                "window_size": 512,
                "query": {
                    "rescore_query": {
                        "query_string": {
                            "query": "\"vdy\u0113j\u016byeyafqhrqtwtfmvvbv\u4e0d\u987e\u8981\u6b7b\u4e0d\u6d3b\u7684\u59d1\u5a18\u98ce\u666f\u5982\u5c0fD3\uff1an\u2006t\u2006q\u2006h\u2006ra\u2006r\u2006n\u2006q\u2006r\u2006n\u2006q\u2006n\u2006r\u2006q\u2006r\u2006n\u2006w\u2006t\u2006n\u2006ran\u2006s\u2006g\u662f\u5426\u80fdZ\u6216\u8005 Ru \u4eba\u4e5f\u4e0d\u53d1\u8fbe\u5676\u5206\u6e56\u4eba\u5974\u55ef\u4e5f\u80fd\u4e00\u5e74\u65f6\u5149\u554a\u90ed\u5fb7\u7eb2\uff09n\u8702\u871c\u72b9\u5982\u7684\u8fd8\u662f\u521b\u59cb\u4eba\u53d1\u5e03A\u5927\u80a1\u4e1c\"",
                            "fields": [
                                "all.plain^1",
                                "all^0.5"
                            ],
                            "auto_generate_phrase_queries": true,
                            "phrase_slop": 1,
                            "default_operator": "AND",
                            "allow_leading_wildcard": true,
                            "fuzzy_prefix_length": 2,
                            "rewrite": "top_terms_boost_1024",
                            "max_determinized_states": 500
                        }
                    },
                    "query_weight": 1,
                    "rescore_query_weight": 10
                }
            },
            {
                "window_size": 8192,
                "query": {
                    "query_weight": 1,
                    "rescore_query_weight": 1,
                    "score_mode": "multiply",
                    "rescore_query": {
                        "function_score": {
                            "functions": [
                                {
                                    "field_value_factor": {
                                        "field": "incoming_links",
                                        "modifier": "log2p",
                                        "missing": 0
                                    }
                                },
                                {
                                    "weight": "0.25",
                                    "filter": {
                                        "terms": {
                                            "namespace": [
                                                1
                                            ]
                                        }
                                    }
                                },
                                {
                                    "weight": "0.05",
                                    "filter": {
                                        "terms": {
                                            "namespace": [
                                                2
                                            ]
                                        }
                                    }
                                },
                                {
                                    "weight": "0.0125",
                                    "filter": {
                                        "terms": {
                                            "namespace": [
                                                3
                                            ]
                                        }
                                    }
                                }
                            ]
                        }
                    }
                }
            }
        ]
    },
    "options": {
        "search_type": "dfs_query_then_fetch",
        "timeout": "20s"
    }
}